src/libraries/Common/src/Interop/Interop.ICU.cs

```diff
@@ -10,7 +10,16 @@ internal static partial class Interop
     internal static partial class Globalization
     {
         [DllImport(Libraries.GlobalizationNative, EntryPoint = "GlobalizationNative_LoadICU")]
-        internal static extern int LoadICU();
+        private static extern int LoadICUDirect();
+
+        [DllImport(Libraries.GlobalizationNative, EntryPoint = "GlobalizationNative_LoadICUData")]
+        private static extern int LoadICUData(string? path);
+
+        internal static int LoadICU()
```

Comment:
The convention is to only have the interop definitions in the `Interop.*` files. The additional logic should be elsewhere.